### PR TITLE
Vera

### PR DESCRIPTION
**Pull request by codegod100**

### CODE REVIEW

File: app/templates/nodes.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h1> {{header}} </h1>
 <h1> Nodes </h1>
 {% endif %}
 {% for node in nodes %}
-<a href="{{node.url}}">{{node.wikilink}}</a> ({{node.size()}})<br />
+<a href="{{node.url}}/uprank/foo,bar">{{node.wikilink}}</a> ({{node.size()}})<br />
```
Comment (flancian) [inline app/templates/nodes.html]: I don't get the /foo,bar here, is it a remnant from a test perhaps? :)

**Review by flancian — COMMENTED** [review]
Thank you, this looks great! One nit only.

I think it's better if you sign https://cla.developers.google.com/ also with this account; sorry for the hassle.

File: app/templates/nodes.html (hunk 1 above)
Comment (codegod100) [reply app/templates/nodes.html]: yeah that's cruft, i'll fix

Comment (codegod100) [reply app/templates/nodes.html]: ok updated

**Review by flancian — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**